Common/TableProducer/centralityTable.cxx

```diff
@@ -183,8 +197,14 @@ struct CentralityTable {
     if (estV0M == 1) {
       float cV0M = 105.0f;
       if (V0MInfo.mCalibrationStored) {
-        float v0m = collision.multV0A() * V0MInfo.mhVtxAmpCorrV0A->GetBinContent(V0MInfo.mhVtxAmpCorrV0A->FindFixBin(collision.posZ())) +
-                    collision.multV0C() * V0MInfo.mhVtxAmpCorrV0C->GetBinContent(V0MInfo.mhVtxAmpCorrV0C->FindFixBin(collision.posZ()));
+        float v0m;
+        if (genName->length() != 0) {
```

Comment:
and probably add a bool for this as well, as length will inspect the string on each collision I believe...

Reply:
`length` and `size` on std::string are of constant complexity, i.e. they don't go through the string
But as we are not looping over collisions but getting called on each of them it does make sense to have a single memory access instead of two (on looping over collisions compiler optimization will take care of it)
I will use the formula (or what the final implementation will be) pointer as 'the' bool